src/libPMacc/include/algorithms/math/doubleMath/sqrt.tpp

```diff
@@ -50,7 +50,7 @@ struct RSqrt<double>
 
     HDINLINE double operator( )(const double& value )
     {
-#if defined(_MSC_VER) && !defined(__CUDA_ARCH__)
```

Comment:
As the code also controlled it before: since GCC supports a `rsqrt` on the host side, one could check for either NVCC or GCC to use the intrinsic.